render/action.yml

```
@@ -0,0 +1,19 @@
+name: 'Render Quarto project'
+author: 'Carlos Scheidegger'
+description: 'This action will render an existing quarto project'
+inputs:
+  to:
+    description: 'Format to render output. Use "all" to generate all outputs, when eg rendering a book in HTML and PDF'
+    required: false  
+runs:
+  using: 'composite'
+  steps: 
+    - name: 'Render'
+      run: |
+        quarto render
```

Should probably be like this, shouldn't it ?
```suggestion
quarto render --to ${{ inputs.to }}
````

---

You're right, we're missing a treatment here, thank you for catching it. (Your solution doesn't quite work because inputs can be empty and there seems to be no way to distinguish between missing and empty values https://github.com/actions/runner/issues/924 https://github.com/actions/toolkit/issues/940) Nevertheless, we need _some_ solution for this, or we should remove that `to` option.